src/pages/version/downstreamTasks/DownstreamProjectAccordion.tsx

```diff
@@ -172,17 +179,14 @@ export const DownstreamProjectAccordion: React.VFC<
         defaultOpen={status === PatchStatus.Failed}
         title={variantTitle}
         titleTag={FlexContainer}
```

Comment:
It might look a little cleaner &  more true to Lara's design if we put the extra content inside of the Accordion (as like a `subtitle` or `description`)

https://user-images.githubusercontent.com/47064971/226731687-8403ad61-29ee-43b5-81c3-6cdfe0f34f7f.mov

Reply:
Done added a subtitle field